git_trac/releasemgr/app.py

```diff
@@ -131,7 +131,11 @@ def merge(self, ticket_number, close=False, allow_empty=False,
         self.git.echo.fetch('trac', branch)
 
         print('Merging ticket...')
-        self.git.echo.merge('FETCH_HEAD', '--no-ff', '--no-commit')
+        try:
+            self.git.echo.merge('FETCH_HEAD', '--no-ff', '--no-commit')
+        except GitError:
+            # Merge conflicts are recognized below
+            pass
```

Comment:
Exit code should be non-zero in case of a merge conflict (both the merge and merge-all subcommands)

Merge conflicts are **NOT** recognized below (maybe in merge-all but not in merge)

Reply:
Yes they are, in `raise ValueError('merge was not clean: conflicts in {}'.format(conflicts))`.

Reply:
That it didn't have the correct exit code was caused by the strange error handling in `cmdline.py`. Fixed now in d37b37c.